recipes/conch_sounds/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "conch_sounds" %}
+{% set version = "0.3.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/conch-sounds-{{ version }}.tar.gz
+  sha256: 8dffbbf154baec8f265eaa0205b26f6edfcf0a39f94bb588cc540c8aa244fe30
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - pytest
+    - python >=3.6
+    - librosa >=0.8.0,=0.8.*
+    - pyraat
+  run:
+    - librosa >=0.8.0,=0.8.*
```

Comment:
I cannot find the version upstream: https://github.com/mmcauliffe/Conch-sounds/blob/7668612d7a610d0f5ae3332f990e71b26c5e8b34/setup.py#L63 

Is this listed somewhere else?

Reply:
I pinned the librosa dependency to >= 0.8.0 due to build errors when testing locally. There was an update to a librosa subdependency which now breaks builds from earlier versions: https://github.com/numba/numba/issues/5865 https://github.com/librosa/librosa/pull/1107

The dependency should be pinned upstream as well (though it isn't yet).